tests/e2e/controlplane/control_plane_test.go

```diff
@@ -274,6 +274,10 @@ spec:
 			To(Succeed(), "Operator failed to be deleted")
 		GinkgoWriter.Println("Operator uninstalled")
 
+		if ocp {
+			Success("Skipping deletion of operator namespace to avoid removal of operator container image from internal registry")
```

Comment:
Here, we should use GinkgoWriter instead of Success because Success is intended only for logging successful steps. The idea is to clearly indicate to anyone reading the test—and not just the result—that Success is a success step in the test case, and should not be used for general information or debugging purposes.